examples/benzeneselenol.cjson

```diff
@@ -22,26 +22,85 @@
+                       1510.1919  1611.9088  1619.0156  2391.2487  3165.1596  3171.3909
+                       3182.0753  3188.5786  3198.0359 ]
+    "symmetry": [  u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
+                   u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
+                   u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A' ]
+    "displacement": [  [  [ 0.    0.   -0.06]
+                          [ 0.    0.   -0.02]
+                          [ 0.    0.    0.03]
+                          ...,
+                          [-0.01  0.   -0.01]
+                          [ 0.    0.   -0.03]
+                          [-0.01 -0.07  0.97]]
+
+                       [  [ 0.    0.01  0.19]
+                          [ 0.    0.    0.03]
+                          [ 0.   -0.01 -0.2 ]
+                          ...,
+                          [-0.02  0.    0.11]
+                          [ 0.    0.    0.04]
+                          [-0.04 -0.04  0.73]]
+
+                       [  [ 0.03 -0.11  0.01]
+                          [-0.13 -0.01  0.  ]
+                          [-0.12  0.2  -0.01]
+                          ...,
+                          [ 0.32 -0.09  0.  ]
+                          [-0.03 -0.08  0.  ]
+                          [ 0.37 -0.14  0.15]]
+
+                          ...,
+                       [  [-0.04  0.    0.  ]
+                          [-0.01  0.02  0.  ]
+                          [-0.01 -0.03  0.  ]
+                          ...,
+                          [-0.24 -0.43  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]
+
+                       [  [-0.02  0.    0.  ]
+                          [ 0.02 -0.03  0.  ]
+                          [-0.03 -0.06  0.  ]
+                          ...,
+                          [ 0.21  0.37  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]
+
+                       [  [-0.06  0.    0.  ]
+                          [-0.02  0.04  0.  ]
+                          [ 0.02  0.02  0.  ]
+                          ...,
+                          [ 0.2   0.37  0.  ]
+                          [ 0.    0.    0.  ]
+                          [ 0.    0.    0.  ]]]
+    "intensities": [  3.1499	3.2259	0.3196	0.0386
+                      0.0307	9.8302	0.0949	10.3548
+                      14.2804	56.5224	7.7038	0.0192
+                      0.4721	2.6001	5.1609	0.0025
+                      13.5903	14.2523	3.9936	0.1947
+                      2.1434	1.1767	1.2111	7.4301
+                      26.7973	1.4953	37.3637	9.7197
+                      2.2009	0.4627	7.5344	18.0167
+                      11.2 ]
   },
   "optimization": {
-    "optdone": True
-    "optstatus": [1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 2 ]
+    "done": True
+    "status": [1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 2 ]
+    "nbasis": 407
```

Comment:
I don't think this would fall under optimization - it's the number of basis functions.